package.json

```diff
@@ -1300,22 +1300,12 @@
                             {
                                 "required": [
```

Comment:
Tasks in this screenshot: incorrect (both), incorrect (neither), correct (file), correct (module)
![image](https://user-images.githubusercontent.com/36966225/127540019-559b1a05-5152-4e48-ab41-030eabd9434b.png)